src/util/security.ts

```diff
@@ -0,0 +1,14 @@
+import bs58 from 'bs58'
+import createHash from 'create-hash'
```

Comment:
These dependencies (buffer, bs58, and create-hash) aren't in edge-core-js, so adding them will increase our bundle size. The good news is that the core already gets the same features from other libraries:

```
import baseX from 'base-x'
import base16 from 'rfc4648'
import hashjs from 'hash.js'

const base58Codec = baseX(
  '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz'
)

export const syncKeyToRepoId = (syncKey: string): string => {
  const bytes = base16.parse(syncKey)
  const hashBytes = sha256(sha256(bytes))
  return base58Codec.encode(hashBytes)
}

function sha256(data: Uint8Array): Uint8Array {
  const hash = hashjs.sha256()
  return Uint8Array.from(hash.update(data).digest())
}
```

Please note that we use `"base-x": "^1.0.4"` to avoid the 100K `Buffer` polyfill, which is a horrible plague upon our ecosystem.